src/main/java/io/github/lokka30/phantomcombat/PhantomCombat.java

```diff
@@ -46,6 +48,9 @@ public void onEnable() {
         log(LogLevel.INFO, "----- ENABLING BEGAN ----");
         log(LogLevel.INFO, "&8[&71/5&8] &7Checking compatibility...");
         checkCompatibility();
+        if (Bukkit.getPluginManager().getPlugin("TempFly") != null) {
+        	try {tempflyRequirement = new TempFlyRequirement();} catch (Exception e) {}
```

Comment:
I put this in a try block as it will error with old TempFly versions that do not contain requirement provider. I didn't factor in their support as it was bugged anyway. You can add a message here in console to update TempFly to latest version for support.